python/pyarrow/__init__.py

```diff
@@ -56,6 +56,8 @@
                             FloatValue, DoubleValue, ListValue,
                             BinaryValue, StringValue)
 
+import pyarrow.schema as _schema
```

Comment:
this was pretty annoying, after `from pyarrow.schema import schema` the statement `import pyarrow.schema` returns `pyarrow.schema.schema` instead of the module. Other ideas?